### DISCUSSION TITLE
Which configuration parameters are used to adjust method chaining styling?

### DISCUSSION THREAD

**Discussion by akveton** [uncrustify/uncrustify#4385] [Q&A]
For example:
```
QString result = QString("Hello %1").arg("World").arg(42);
```
vs

```
QString result = QString("Hello %1")
                     .arg("World")
                     .arg(42);
```
                     
Which parameters in Uncrustify can be used to adjust the styling of method chaining? I looked pretty hard, but could not find anything related to this in the "nl" section (or any other sections, for that matter).

**Comment by mmichalek-ibl** [comment]
Hello,
maybe `indent_member` ?
